docs/guides/setup/setup_overview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Setup: Overview"
+---
+# [![Setup Icon](../../images/universal_map/Gear-active.png)](setup_overview.md) Setup: Overview 
+
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='active' connect='inactive' create='inactive' validate='inactive'/>
+
+<!-- Only keep one of the 'To best understand this document' lines.  For processes like the Universal Map steps, use the first one.  For processes like the Architecture Reviews, use the second one. -->
+
+:::note Prerequisites
+- To best understand this document, you must have completed Step 1: Setup of the [Getting Started tutorial](../../tutorials/getting_started/initialize_a_data_context.md).
+:::
+	
+Getting started with Great Expectations is quick and easy.  Once you have completed your setup, you will have access to all of the features of Great Expectations from a single entry point: Your Data Context.  You will also have your Metadata Stores and Data Docs configured in the manner most suitable for your project's purposes.
```

Comment:
Style Guide question: "Metadata Stores" or "Stores"?

We don't actually have an abstraction called MetadataStore, but we do have Store. OTOH, "Metadata Store" is a good way to let people know that we're talking about a specific technical thing.

Reply:
A decision on this would be really helpful. I run into it all the time in our docs.